### ISSUE TITLE
Displaying Hero Stats

### ISSUE THREAD

**Issue by darklordabc** [ash47/LegendsOfDota#259]
I'm making this issue for the sake of clarity as this has been already discussed in another issue.

There is many differences between heroes that give knowledgeable players unfair advantages. Such as which heroes have highest move speed, most str gain per level, furthest attack range etc. Some heroes also have unique and strange interactions with abilities and game mechanics that new players would have no way of knowing, e.g. Alchemist giving aghs ability is only possible on alchemist hero and is unrelated to Chemical Rage.

The suggested fix is to provide the most important stats on the hero select screen, like how LoD on dota 1 does. These are two mockups of how the info could be displayed. I imagine these popups appear with mouse over, like how when u mouse over an ability the tooltip will popup. (Keep in mind that these are mockups, and the text, colors and fonts are all not final).
![image](https://cloud.githubusercontent.com/assets/16277198/14967392/7fdaa666-10fa-11e6-9479-fffd1917da07.png)
![image](https://cloud.githubusercontent.com/assets/16277198/14967400/8c2581fc-10fa-11e6-9d9f-773c4a96341b.png)
And finally this is how it could look ingame:
![image](https://cloud.githubusercontent.com/assets/16277198/14967445/d07fb17e-10fa-11e6-9920-db4ebf022204.png)

**Comment by Jazzer008** [comment]
If [issue #257](https://github.com/ash47/LegendsOfDota/issues/257) is resolved then I could be assigned to this.

**Comment by Jazzer008** [comment]
@ash47 Missing some attributes from hero info in panorama game_setup.js?

![Hero Stats Preview with Missing Data](http://i.imgur.com/dqLjLV2.png "With Missing Data")

There's actually a panorama event called DOTAShowHeroStatsTooltip but it's not available in the events dump, there's no documentation online and only a [single mention](https://www.google.co.uk/webhp?sourceid=chrome-instant&ion=1&espv=2&ie=UTF-8#q=DOTAShowHeroStatsTooltip) I could find across the internet.

From the console errors I know it could potentially be called and it takes 3 arguments not including the panel, but without any documentation I have no idea what it might need, or whether it's actually possible for us to use that type of Tooltip outside of the actual game scene.

Also, it seems that the Tooltips really don't want to take much in the way of css styling. From what I could tell they can only handle the html font color tag. There are DOTAShowTooltip'Styled' events that supposedly take a string css class name but users report this isn't accessilbe/functional for custom addon authors.

**Comment by darklordabc** [comment]
@ash47, sorry to be bombarding you with notifications, but this feature is also basically completed. Just waiting on feedback.

**Comment by ash47** [comment]
This has been implemented in master.

**Comment by darklordabc** [comment]
Great work, it looks great. Can I make one suggestion? Can we group the more relevant stats together, or maybe have them a different color. By relevant I mean the stats that vary more, like movement speed, range, attributes etcs.
